Sources/AuthgearError.swift

```diff
@@ -25,6 +25,7 @@ public enum AuthgearError: CustomNSError, LocalizedError {
     case publicKeyNotFound
     case error(Error)
     case runtimeError(String)
+    case notAllowed(String)
```

Comment:
Rename this to preAuthenticatedURLNotAllowed

Reply:
Renamed in later commits.